picturae_importer.py

```diff
@@ -1044,8 +1049,8 @@ def run_all_methods(self):
         self.logger.info("process finished")
 
         # unlocking database
```

Comment:
locking and unlocking the user does not seem to work with the priveleges set on the real maria db , with a dedicated importing agent ID, to be attached to all records this no longer seems necessary ?